node/cn/handler.go

```diff
@@ -258,8 +258,8 @@ func NewProtocolManager(config *params.ChainConfig, mode downloader.SyncMode, ne
 }
 
 // istanbul BFT
```

Comment:
What is the meaning of this comment?

Reply:
I think that this means this is not something generally used method, but the method for Istanbul BFT. If we don't use Istanbul BFT, we don't need to manage the list of validators.